kruize-clowdapp.yaml

```diff
@@ -101,6 +101,8 @@ objects:
             value: ${KRUIZE_CW_LOG_STREAM}
           - name: logging_cloudwatch_logLevel
             value: ${KRUIZE_CW_LOGGING_LEVEL}
+          - name: plots
+            value: "true"
```

Comment:
We'll need to grab the env var in kruize-clowdapp.

```
          - name: plots
            value: ${PLOTS_DATA}

- name: PLOTS_DATA
  value: "false"
```
The "false" should get overridden through deployment repo params.

Reply:
Changes pushed